index.html

```diff
@@ -0,0 +1,781 @@
+                        <li>Kelder van ca. 22 m aanwezig</li>
+                        <li>Alarmsysteem aanwezig</li>
+                        <li>Waterontharder aanwezig</li>
+                    </ul>
+                      
+                    <p>Recent onderhoud/werkzaamheden:</p>
+                    <ul>
+                        <li>Vloer geïsoleerd 2020</li>
+                        <li>Hemelwaterafvoeren vernieuwd</li>
+                        <li>Algeheel schilderwerk buiten, grotendeels 2023</li>
+                        <li>Zijgevel woning gereinigd en geïmpregneerd 2023</li>
+                        <li>Dakpannen gereinigd 2023</li>
+                        <li>Nieuwe kap op schoorsteen, cement aangeheeld (2022)</li>
+                        <li>Vernieuwde cv-ketel (2019), voorbereid voor hybride warmtepomp</li>
+                        <li>Dakplaten garages vervangen</li>
+                    </ul>
+                </div>
+                
+                <label class="readMore txt"><input name="volledige omschrijving" type="checkbox"></label>
+                
+            </section>
+
+
+            <section class="ing imageRight">
+                <div>
+                    <h3>Kan je dit huis betalen?</h3>
+                    <p>Bereken nu direct hoeveel u kunt lenen. <a href="#">Bereken nu</a></p></div>
+                <div>
+                    <img src="./assets/icons/ING.svg" alt="logo van de bank ING">
+                </div>
+            </section>
+
+
+            <section class="kenmerken">
+                <h2>Kenmerken</h2>
+                
+                <h4>Overdracht</h4>
+                <dl>
+                    <dt>Vraagprijs</dt>
+                    <dd>€ 750.000 kosten koper <span><a href="https://www.funda.nl/hypotheek-advies/7011148/">Hypotheekadvies</a></span></dd>
+
+                    <dt>Vraagprijs per m²</dt>
+                    <dd>€ 3.827</dd>
+                    
+                    <dt>Aangeboden sinds</dt>
+                    <dd><button class="txt">Log in om te bekijken</button></dd>
+                    
+                    <dt>Status</dt>
+                    <dd>Beschikbaar</dd>
+                    
+                    <dt>Aanvaarding</dt>
+                    <dd>In overleg</dd>
+                </dl>
+
+
+                <h4>Bouw</h4>
+                <dl>
+                    <dt>Soort woonhuis</dt>
+                    <dd>Eengezinswoning, 2-onder-1-kapwoning</dd>
+                    
+                    <dt>Soort bouw</dt>
+                    <dd>Bestaande bouw</dd>
+                    
+                    <dt>Bouwjaar</dt>
+                    <dd>1977</dd>
+                    
+                    <dt>Soort dak</dt>
+                    <dd>Zadeldak bedekt met pannen</dd>
+                </dl>
+
+                <h4>Oppervlakten en inhoud</h4>
+                <dl>
+                    <dt class="eenKolom">Gebruiksoppervlakten</dt>
```

Comment:
Desondanks mijn eerste reactie over consistentie naamgeving, zou ik niet kiezen voor classes in meerdere talen.